simple_salesforce/api.py

```diff
@@ -59,8 +59,9 @@ class Salesforce(object):
     def __init__(
             self, username=None, password=None, security_token=None,
             session_id=None, instance=None, instance_url=None,
-            organizationId=None, sandbox=False, version=DEFAULT_API_VERSION,
-            proxies=None, session=None, client_id=None):
+            organizationId=None, version=DEFAULT_API_VERSION,
+            proxies=None, session=None, client_id=None, login_domain='login',
+            **kwargs):
```

Comment:
Can you remove `kwargs` and replace it with `sandbox` for the time being, keeping the same deprecation warning in place?